.github/workflows/lza_ci_infrastructure.yml

```diff
@@ -0,0 +1,60 @@
+name: LZA Continuous Integration Infrastructure
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - "main"
+    paths:
+      - "lza/**"
```

Comment:
Hi Ian, I wonder if this path should be `lza*/**`?

Reply:
The path is updated.